content/aws/iam/assuming-roles.md

```diff
@@ -7,3 +7,32 @@ tags:
   - aws
   - assume-role
 ---
+
+# Assume Role via AWS Web Console
+
+**NOTE**: Due to the security implications, IAM policies are set up to **only** allow the root AWS account to assume roles into other accounts.
+
+1. Log into the AWS root acccount
+{{< img src="/assets/aws-root-login.png" title="Example AWS root login" >}}
+
+2. Click on `user@example.com @ example-root-aws` drop down at the top of the console and select `Switch Role`
+  - Enter the AWS Account ID of the account to assume in `Account` field
```

Comment:
simplify?
Enter the ID of the member account in `Account` field

Reply:
👍